libpod/runtime_ctr.go

```diff
@@ -263,3 +267,22 @@ func (r *Runtime) GetContainersByList(containers []string) ([]*Container, error)
 	}
 	return ctrs, nil
 }
+
+// GetLatestContainer is a helper function for GetContainers which
+// returns a container object of the latest created container.
+func (r *Runtime) GetLatestContainer() (*Container, error) {
+	var lastCreatedIndex int
+	var lastCreatedTime time.Time
+	ctrs, err := r.GetAllContainers()
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to find latest container")
+	}
+	for containerIndex, ctr := range ctrs {
+		createdTime := ctr.config.CreatedTime
+		if createdTime.After(lastCreatedTime) {
+			lastCreatedTime = createdTime
+			lastCreatedIndex = containerIndex
+		}
+	}
+	return ctrs[lastCreatedIndex], nil
+}
```

Comment:
Would it be possible to create a db function to do something like:
SELECT MAX(createDate) from ContainerTable?

And call that?  I'm just concerned with 1,000's of containers this routine might get a bit on the laggy side.

Reply:
Won't work once we swap to boltdb later this week - our queries there are a lot more limited, so we can't really filter until we've already retrieved a full container

Reply:
yeah we discussed this before i set out to do it

Reply:
I spent a bit of time reading up on BoltDB this morn.  I'm a bit concerned to see how it will handle a large environment, especially chunks of code like this.  It would be good if we can plan to set up some stress tests sooner rather than later with a large number of containers/images in play.

Regardless, thanks for the 411.